client/src/components/CreateRoseBudThorn.jsx

```diff
@@ -0,0 +1,13 @@
+
+export default function CreateRoseBudThorn() {
+  return (
+    <div classname="create" >
+
+    <h2>Create a new RoseBudThorn</h2>
+
+    </div>
+  )
+}
+
+//learned:
+//sfc tab
```

Comment:
sfc tab?

Reply:
In VScode, if you create a jsx file, instead of manually typing up the basic code to set up a new component, you can type "sfc" in the blank jsx file and hit tab.  This will populate the component with the basic set up.  It is a cool trick I learned.   I will delete it for now, and write in in a "shortcuts" note.